purchases/src/test/java/com/revenuecat/purchases/paywalls/components/TextComponentTests.kt

```diff
@@ -12,28 +12,32 @@ import com.revenuecat.purchases.paywalls.components.properties.Size
 import com.revenuecat.purchases.paywalls.components.properties.SizeConstraint
 import org.intellij.lang.annotations.Language
 import org.junit.Test
+import org.junit.experimental.runners.Enclosed
 import org.junit.runner.RunWith
 import org.junit.runners.Parameterized
 
-@RunWith(Parameterized::class)
-internal class TextComponentTests(@Suppress("UNUSED_PARAMETER") name: String, private val args: Args) {
```

Comment:
Reviewing this file using GitHub's "Hide whitespace" option makes it much clearer what has actually changed.